pom.xml

```diff
@@ -193,7 +193,7 @@
             </activation>
             <properties>
                 <build.number>Milestone Build</build.number>
-                <online.repo>${oh.repo.releaseBaseUrl}/online-repo-milestone/${online.repo.version}</online.repo>
+                <online.repo>https://openhab.jfrog.io/openhab/libs-milestone</online.repo>
```

Comment:
What's the rationale for changing the repo URLs? It renders the [properties from the super POM](https://github.com/openhab/infrastructure/blob/f0d31d47a3b5819900336ca30777c3c6f70877f8/pom.xml#L54-L59) to be rather useless.

Reply:
Well, the main reason being that it was already changed for the Release builds with https://github.com/openhab/openhab-distro/pull/1014, so now the snapshot&milestone urls are in line.

But yes, for the master branch, we should best get it back to use properties of the super pom.